torch/_inductor/config.py

```diff
@@ -400,6 +401,33 @@ class triton:
     inject_relu_bug_TESTING_ONLY = None
 
 
+class cuda:
+    # CUDA arch to use for CUDA template kernel compilation.
+    # e.g. "70", "75", "80", "90", etc.
+    # When arch is None, Inductor uses torch.cuda.get_device_capability(0).
+    arch = None
+
+    # CUDA version to use for CUDA template kernel compilation.
+    # e.g. "11.4", "12.1", etc.
+    # When version is None, Inductor uses torch.version.cuda.
+    version = None
+
+    # Optimization level for the host compiler.
+    compile_opt_level = "-O1"
```

Comment:
`-O1` looks a bit strange here as a default. Shouldn't we have either `-O0` (for no optimization) or `-O3` (for max. optimization) by default?